.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+	"name": "Rust",
+	"image": "mcr.microsoft.com/devcontainers/rust:latest",
+	"runArgs": ["--platform=linux/amd64"]
+}
```

Comment:
Is this used anywhere?

Reply:
no, it's for dev purpose. so when you start devcontainer or github codespaces it will be ubuntu-rust linux/x86